src/shared/condition.c

```diff
@@ -1134,7 +1136,9 @@ static int condition_test_psi(Condition *c, char **env) {
                 return log_debug_errno(r, "Failed to parse loadavg: %s", c->parameter);
 
         r = read_resource_pressure(pressure_path, PRESSURE_TYPE_FULL, &pressure);
-        if (r == -ENODATA) /* cpu.pressure 'full' was added recently, fall back to 'some'. */
+        /* cpu.pressure 'full' is undefined at system level since 5.13,
+         * but remains reported and set to 0 for compatibility. Fall back to 'some' */
```

Comment:
hmm, was the "full" line ever useful before kernel 5.13? If it always was a mistake and not useful, maybe we should never look into it for the system wide stuff?

Reply:
Looking into it, it appears to have been used and reported values prior to 5.13. But according to the [commit](https://github.com/torvalds/linux/commit/890d550d7dbac7a31ecaa78732aa22be282bb6b8) where it was removed, it was never of any use.

Reply:
Then it's fine to never use it, no need for the fallback